specs/olm.spec

```diff
@@ -48,4 +48,9 @@ Importance: Critical
 Uninstalls `openshift-pipelines` operator using olm
 Steps:
   * Validate Operator should be installed
-  * Uninstall Operator
\ No newline at end of file
+  * Uninstall Operator
+
+## Verify that Pipelines operator installs two quickstarts 
```

Comment:
I would omit word `two` so that we don't have to updated spec and step in the case of a change in the future